source/user/installation_guide.md

```diff
@@ -44,14 +44,12 @@ In that case, you will [Fork][git guide] the [PowerSimData][powersimdata_github]
 Before using the scenario framework, you need to update the path to the variables in ***PowerSimData/powersimdata/utility/server_setup.py*** file. Note that the **LOCAL_DIR** does not need to be updated if having a **ScenarioData** directory in your home folder is a good place to store scenario data. You can now create, execute, extract and analyze scenarios. The PowerSimData [README][powersimdata_local] goes over these steps.
 
 
-## 3. Validate your Installation
-Make sure the [tox] package is installed and run from the root folder:
-```bash
-pip install tox
+## 3. Using docker
+This option enables running the full framework in containers, colocated on the same machine.
+In this case, the only system requirement is to have docker, (including docker-compose) installed. 
```

Comment:
should we add a link to docker here too?

Reply:
Guess it wouldn't hurt, I could link to this landing page? https://docs.docker.com/get-docker/

Reply:
Looks good